modules/animationqt/src/animationeditordockwidgetqt.cpp

```diff
@@ -268,11 +299,69 @@ AnimationEditorDockWidgetQt::AnimationEditorDockWidgetQt(
     }
 
     toolBar->addSeparator();
+
     controller_.AnimationControllerObservable::addObserver(this);
 }
 
 AnimationEditorDockWidgetQt::~AnimationEditorDockWidgetQt() = default;
 
+void AnimationEditorDockWidgetQt::openAnimation() {
+    InviwoFileDialog openFileDialog(this, "Import Animation ...", "animation");
+    openFileDialog.addSidebarPath(PathType::Workspaces);
+    openFileDialog.addExtension("inv", "Inviwo File");
```

Comment:
add a specific extension for animation files?

Reply:
otherwise, an animation `.inv` can be confused with a regular workspace. What happens then?

Reply:
That is fine, it imports the saved Animation in that workspace.
Export is worse because we cannot open the generated workspace. 
I guess we could add ".ainv"? Might become a bit confusing?

Reply:
Should this even be allowed considering hat it is not self contained. I rather think one would just save the whole workspace and then import the tf

Reply:
If exporting is not allowed we should consider adding support for multiple animations in a workspace. That would be more work, but certainly an option.
Might be a minor issue but the entire workspace is in one of my cases 300 kB and just the animation is 2 kB

Reply:
Don't think the size is anything we have to care about at this point...

Reply:
One could also do export/import on a track level. that those files would then not refer to any other properties not in the file... copy/paste for track contents would also work here I think

Reply:
I generally think saving the animation by them selfs is a very bad idea given the very unclear and invisible requirements for begin able to load them again... would more or less only work when loading them into the same workspace as where is was save from.

Reply:
> would more or less only work when loading them into the same workspace as where is was save from

Agree, that is the use case I have :) So being able to export just this information is useful in that case. In general it might not be as useful. Having support for multiple animations in the workspace would solve this I think.